src/main/java/com/eppo/sdk/helpers/Shard.java

```diff
@@ -16,9 +16,9 @@ public class Shard {
      *
      * @param input
      * @return
-     * @throws Exception
+     * @throws RuntimeException
      */
-    public static String getHex(String input) {
+    public static String getHex(String input) throws RuntimeException {
```

Comment:
`RuntimeException` is not a checked exception, so I don't think it should be part of the method signature. Same with the other places it is added in EppoClient.java.

Reply:
Thanks, removed the `throws RuntimeException`